### ISSUE TITLE
searching string with dot not working

### ISSUE THREAD

**Issue by sturze** [fooplugins/FooTable#414]
Hey Guys,

i have a Table with persons and room numbers. if i search for a room number (cell in example below) i typed 1.15 into the seachfield without result!

```
<td style="display: table-cell;">1.15</td>

```

changing the search string to "1.15" 
(with " .." in the searchbox)
![search](https://cloud.githubusercontent.com/assets/5389555/13346470/c5361844-dc66-11e5-905e-76d2cbb53e08.PNG)

it works !?

**Comment by steveush** [comment]
Hi,

The issue here is that a period (.) is being seen as a phrase connector, along with any dash (-), underscore (_) or plus (+) characters. This is then replaced with a space in the final query executed against the rows so it is looking for "1 15" and not 1.15. By wrapping it in quotes you are specifying an exact match so the period is not replaced and a result is returned. I'll look at adding in an option to disable this behavior or make it search for both "1 15" and 1.15.

Thanks

**Comment by sturze** [comment]
that would be cool! thanks!

**Comment by steveush** [comment]
Hi,

You can now disable the phrase connectors in the latest version (3.0.6), please have a look at the [updated docs](http://fooplugins.github.io/FooTable/docs/components/filtering.html#connectors).

Thanks

**Comment by sturze** [comment]
not working ... :disappointed: 

i just tried with the new version and the tried 

```
<table class="table" data-filtering="true" data-sorting="true" data-filter-connectors="false" 
```

and also switchend it to true, no difference.... 

what did i do wrong?

##edit: 
i just re-read the whole description: it says it replaces connectors, but i dont want to?!
the value is 1.19 and i dont want to search for "1 19" i want to search for "1.19"... i think thats the problem, or am i wrong?!

![img 1](https://cloud.githubusercontent.com/assets/5389555/13598786/0bd25878-e520-11e5-8d33-978ecc21452f.PNG)
![img 2](https://cloud.githubusercontent.com/assets/5389555/13598785/0bd2526a-e520-11e5-8e78-7082d854a5b7.PNG)

**Comment by steveush** [comment]
Hi,

The issue is there is a small bug when trying to set this option using data attributes, I accidentally left it checking for a number value instead of a boolean one. In the mean time please use [the JavaScript way to set this option](http://fooplugins.github.io/FooTable/docs/components/filtering.html#connectors). I'll push an update to fix the data attribute issue shortly.

Thanks

**Comment by steveush** [comment]
Hi,

I've checked in the fix but have not pushed a new version, I'll wait for a few more changes/fixes before pushing 3.0.7. You will need to compile the project using Grunt to get the new changes right now or simply see what I have changed in the commit and make those changes to your local version.

Thanks
